.vscode/launch.json

```diff
@@ -0,0 +1,27 @@
+{
+    "version": "0.2.0",
+    "configurations": [
+        {
+            "name": "debug server",
+            "type": "cppdbg",
+            "request": "launch",
+            "program": "${workspaceFolder}/tigerbeetle",
+            "args": [ "start", "--addresses=127.0.0.1", "0_0.tigerbeetle.debug" ],
+            "stopAtEntry": false,
+            "cwd": "${workspaceFolder}",
+            "environment": [],
+            "preLaunchTask": "format debug server",
+        },
+        {
+            "name": "debug tests",
```

Comment:
:tada:

Reply:
What do you think?

1. Configure an exhaustive list of all debuggable artifacts we produce (i.e. simulator, fuzz tests, etc.), and then keep that in sync every time we add a new artifact.

2. Just start minimal AS IS, and add new entries as it is needed.

Reply:
Yes, we can configure `launch=attach`, and works like this:

![image](https://github.com/tigerbeetledb/tigerbeetle/assets/5728002/1b5ea1fa-f286-4b5e-8584-1143c946f1f4)